app/dashboard/controller.ts

```diff
@@ -99,7 +99,7 @@ export default class Dashboard extends Controller.extend({
 
         const nodes = yield user.queryHasMany('nodes', {
             embed: 'contributors',
-            filter: filter ? { title: filter } : undefined,
+            filter: filter ? { title: $('<div>').text(filter).html() } : undefined,
```

Comment:
This works, but it's kinda gross that we have to do this. It should be the responsibility of the front-end to encode data for display.

Reply:
If you had your druthers, what would change?